module/Database/view/database/index/index.phtml

```diff
@@ -1,5 +1,5 @@
 <div class="jumbotron">
     <h1><span class="gewis-red">GEWIS</span> database tool</h1>
-    <p>Tijdelijke frontpage</p>
+    <p>🍑 Niet meer tijdelijke frontpage 🍑</p>
```

Comment:
Why is this change needed :sweat_smile:

Reply:
Because nobody else will ever make it non-temporary.